tests/intune/test_index_view.py

```diff
@@ -0,0 +1,185 @@
+from django.urls import reverse
+from django.test import TestCase
+
+from functools import reduce
+import operator
+from django.contrib.auth.models import Permission
+from django.db.models import Q
+from django.utils.crypto import get_random_string
+from accounts.models import APIToken, User
+from django.contrib.auth.models import Group, Permission
+from zentral.contrib.inventory.models import MetaBusinessUnit
+from zentral.contrib.intune.models import Tenant
+
+
```

Comment:
Use https://github.com/zentralopensource/zentral/blob/7245d57d31f0d329a711c8e7d7cee43f02cf5a15/tests/wsone/test_setup_views.py#L14

It will probably solve the test errors